src/aomusicplayer.cpp

```diff
@@ -38,6 +36,7 @@ QString AOMusicPlayer::play(QString p_song, int channel, bool loop,
       newstream = BASS_StreamCreateURL(f_path.toStdString().c_str(), 0, streaming_flags, nullptr, 0);
 
   } else {
+    f_path = ao_app->get_real_path(ao_app->get_music_path(p_song));
     if (f_path.endsWith(".opus"))
       newstream = BASS_OPUS_StreamCreateFile(FALSE, f_path.utf16(), 0, 0, flags);
```

Comment:
Let's see about removing this `opus` exception in the future, as we have already loaded all BASS extensions at this point so that they can be invoked with the normal `StreamCreateFile`.